mautic/mautic
# On updateFilterPositioning, some div's id was unchanged

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 


**Description:**
In segment Filters, when changing operators, the filter value doesn't change or changed value of another filter.

**Steps to reproduce the bug:**

1. Go to Segment -> Add New
2. Switch to filters tab
3. Create at least three filters
4. Re-order the filters so the 3rd one becomes the 2nd
5. Change the operator on one of the field.
6. Values for other filters get changed instead of same filter.


**Steps to test this PR:**

1. Go to Segment -> Add New
2. Switch to filters tab
3. Create at least three filters
4. Re-order the filters so the 3rd one becomes the 2nd
5. Change the operator on one of the field.
6. Values for same filters should be changed.

.
